src/index.ts

```diff
@@ -1,26 +1,63 @@
-import { getTargetOrigin, TestBoxConfig } from "./config";
-import { info } from "./utils/logging";
+import type { TestBoxConfig, LoginHandler } from "./config";
+import { sendMessageToTestBox } from "./messaging";
 import {
-  isValidIncomingTestBoxMessage,
-  sendMessageToTestBox,
-} from "./messaging";
-import { routeMessage } from "./router";
-import { INITIALIZE_REQUEST } from "./messaging/outgoing";
+  INITIALIZE_REQUEST,
+  LOGIN_HANDLER_REGISTERED,
+} from "./messaging/outgoing";
 import { LoginEvent, NavigateEvent } from "./messaging/incoming";
+import { messageEventCallback } from "./message-event";
+import { routeMessage } from "./router";
 
 let tbxStarted = false;
+let isLoginHandlerRegistered = false;
```

Comment:
Dont think we need to store this. As we moved to starting the loginHandler as `undefined`, we can use that to check if it is/is not registered.

Reply:
The purpose of this variable is to check if the registerLoginHandler function has been called more than once, we can't check that with just the loginHandler variable

Reply:
Even if we can I think it's safer to have a variable for this

Reply:
In which way you see this as safer? we can extract a bool from checking the function existance.

Reply:
Ok, I was thinking about a case where `startTestbox` is called with a login handler and then `registerLoginHandler` is called with a new login handler, but I guess this is not intended to happen